src/libasr/codegen/asr_to_c.cpp

```diff
@@ -1203,6 +1227,10 @@ R"(
     }
 
     void visit_ArrayItem(const ASR::ArrayItem_t &x) {
+        if (compiler_options.fast && x.m_value != nullptr) {
+            visit_expr(*x.m_value);
+            return;
+        }
```

Comment:
This is exactly the same block repeated in every node. So what about creating macro to generate this and placing the macro everywhere. 

In fact I would like to call it automatically so that one doesn't have to add this to every node. I don't know if C++ supports a way to always insert a code block in subclass method (other than macro one which one has to do by hand in every node). Anyways, let's leave this automation for future, right now just create a macro and use it everywhere.